client/me/concierge/shared/primary-header.js

```diff
@@ -13,21 +13,21 @@ class PrimaryHeader extends Component {
 		return (
 			<Fragment>
 				<ClosureNotice
-					displayAt="2021-03-28 00:00Z"
-					closesAt="2021-04-04 00:00Z"
-					reopensAt="2021-04-05 06:00Z"
+					displayAt="2022-04-10 00:01Z"
```

Comment:
Let's switch this (and other similar) to `00:00Z` (instead of `00:01Z`). It just seems sloppy when we show the message to the user that support will be closed between `00:01 and 07:00`. I'm pretty sure Krystin put it this way to make sure it's April 10 and not somehow the previous/next day, but we can be sure it's correct :)